table/column.go

```diff
@@ -288,6 +288,10 @@ func (c *Column) CheckNotNull(data types.Datum) error {
 	if mysql.HasNotNullFlag(c.Flag) && data.IsNull() {
```

Comment:
`if (mysql.HasNotNullFlag(c.Flag) || mysql.HasPreventNullInsertFlag(c.Flag)) && data.IsNull() ` and remove line 292 - 294